examples/nucleo_f439zi/can_m2515/project.xml

```diff
@@ -0,0 +1,18 @@
+<library>
+  <extends>modm:nucleo-f439zi</extends>
+  <options>
+    <option name="modm:build:build.path">../../../build/stm32f4_discovery/mcp2515</option>
+  </options>
+  <modules>
+    <module>modm:platform:gpio</module>
+    <module>modm:platform:spi.bitbang</module>
+    <module>modm:platform:spi:1</module>
+    <module>modm:platform:spi:2</module>
```

Comment:
Not used in the example. Remove.

Reply:
`    <module>modm:platform:spi:1</module>` is still used, but rest is gone